internal/handlers/irc/irc.go

```diff
@@ -45,7 +47,9 @@ func (c Client) StartBot(errChan chan<- error, sendMessage func(string)) {
 	c.verbose.LogInfo("Starting up IRC bot...")
 	c.sendToTg = sendMessage
 	c.addHandlers()
-	if err := c.Connect(); err != nil {
+	// TODO: Currently just set to 5 seconds,
+	// if we want this to be configurable we can add a config option
+	if err := c.DialerConnect(&net.Dialer{Timeout: 5 * time.Second}); err != nil {
```

Comment:
I'm struggling to think of whether or not to add this to our config. Personally, I think this should be set to something like 10 seconds to account for any delays IRC side.

Reply:
@Tjzabel Based on our design goals for v2.0.0, I'd prefer to err on the side of simplicity and leave it out, since I think most cases won't need this to be a custom value. If we are wrong and it turns out it needs to be configurable, it shouldn't be that hard to convert later.

Reply:
Do we want to move it to 10 seconds, then?

Reply:
Sure, 10 seconds works for me.